cruise-control-metrics-reporter/src/test/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsReporterSslTest.java

```diff
@@ -43,14 +43,15 @@ public Properties overridingProps() {
         props.put(appendPrefix(configName), value);
       }
     }
-    props.setProperty("metric.reporters", CruiseControlMetricsReporter.class.getName());
-    props.setProperty("listeners", "SSL://127.0.0.1:" + port);
+    props.setProperty(CommonClientConfigs.METRIC_REPORTER_CLASSES_CONFIG, CruiseControlMetricsReporter.class.getName());
+    props.setProperty(KafkaConfig.ListenersProp(), "SSL://127.0.0.1:" + port);
     props.setProperty(CruiseControlMetricsReporterConfig.config(CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG), "127.0.0.1:" + port);
-    props.setProperty(CruiseControlMetricsReporterConfig.config(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG), "SSL");
+    props.setProperty(CruiseControlMetricsReporterConfig.config(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG), SecurityProtocol.SSL.name);
     props.setProperty(CRUISE_CONTROL_METRICS_REPORTING_INTERVAL_MS_CONFIG, "100");
     props.setProperty(CRUISE_CONTROL_METRICS_TOPIC_CONFIG, TOPIC);
     props.setProperty(KafkaConfig.LogFlushIntervalMessagesProp(), "1");
     props.setProperty(KafkaConfig.OffsetsTopicReplicationFactorProp(), "1");
```

Comment:
any particular reason we set  `OffsetsTopicReplicationFactorProp` to 1 here?
 since we change the cluster size to 2 in `CCKafkaIntegrationTestHarness` and set `DefaultReplicationFactorProp` to 2, I feel we can remove this one.

Reply:
Setting the replication factor for the `offsets` topic to `1` was actually useful for our test case, because during the rebase, it helped us explicitly verify that a subset of MBeans (e.g. `REQUESTS_PER_SEC` > `FOLLOWER_FETCH_REQUEST_TYPE`) are not generated in Kafka 2.0 unless we have a topic with follower that we produce data to.